robupy/__init__.py

```diff
@@ -1,6 +1,6 @@
-import warnings
 import socket
+import warnings
 
 # This turns of the RuntimeWarnings during production so that they do not clutter the log files.
```

Comment:
This is too machine-specific. Just remove at this point.

Reply:
Okay. Will do that in the next PR when I include Codecov badge!